rules/library/resources.bzl

```diff
@@ -4,6 +4,23 @@ def _is_apple_resource_file(file, extensions_to_filter):
 
     return True
 
+# Remvoe files whose parent folders are already in the list. E.g., when both "foo/" and "foo/bar"
+# are in the files list, we should only return "foo/". This de-duplication is needed after this
+# rules-apple PR: https://github.com/bazelbuild/rules_apple/pull/1311. Otherwise 'clonefile' will
+# raise exceptions when both "foo/" and "foo/bar" are copied.
+def _deduplicate_paths(files):
```

Comment:
the name does not match with implementation in my opinion... maybe sth like `_most_common_parent_dirs` for files which then means in extreme case, return just one dir if all files under same root dir?

Reply:
make sense, let me do the renaming

Reply:
fixed.